alerts/cluster-monitoring-operator/PrometheusScrapeBodySizeLimitHit.md

```diff
@@ -0,0 +1,105 @@
+# PrometheusScrapeBodySizeLimitHit
+
+## Meaning
+
+The alert `PrometheusScrapeBodySizeLimitHit` is triggered when at least one of
+Prometheus' scrape targets replies with a response body larger than `body_size_limit`.
+
+## Impact
+
+The metrics coming from the endpoint will not be scraped each time when
+the response body size exceeds the body size limit.
+
+## Diagnosis
+
+First we need check the value of `body_size_limit` in the Prometheus configuration
+file, which is saved in the secret `prometheus-k8s` in the namespace `openshift-monitoring`.
+Search for the field `body_size_limit`.
```

Comment:
I would rather get the value from the Prometheus object.

```
oc get prometheus k8s -o jsonpath='{.spec.enforcedBodySizeLimit}'
```

Reply:
The value is in the configmap `cluster-monitoring-config`, I will add how to find out the bodysize limit from the configmap, too.